config/packages/security.yaml

```diff
@@ -19,6 +34,14 @@ security:
 
     # Easy way to control access for large sections of your site
     # Note: Only the *first* access control that matches will be used
+
+
+    # custom add US_05 exemple , TODO more role and hierarchy will be add ROLE_ENTERPRISE / ADVISOR ...
+    role_hierarchy:
+        ROLE_ADVISOR:   ROLE_USER
+        ROLE_ENTREPRISE:    ROLE_USER
+        ROLE_ADMIN: [ROLE_USER,ROLE_ADVISOR,ROLE_ENTREPRISE]
```

Comment:
Remove `ROLE_ADVISOR` and `ROLE_ENTERPRISE` from `ROLE_ADMIN` (to be directly handled in 2nd twig condition : `if is_granted(ROLE_ENTERPRISE) or is_granted(ROLE_ADMIN)`